src/corehost/cli/deps_resolver.cpp

```diff
@@ -164,6 +164,18 @@ void deps_resolver_t::setup_shared_store_probes(
     }
 }
 
+pal::string_t deps_resolver_t::get_probe_directories()
+{
+    pal::string_t directories;
+    for (const auto& pc : m_probes)
```

Comment:
> but i would rather not put more logic in the host

I disagree - we need to put the right logic in the host, not just what gets everything. Otherwise, we will run into support scenarios where we will be forced to fix issues when people start to take dependence on this and when certain paths wont make sense. For example, even if you pass everything to the API user, they have no way of knowing which can be used in a nuget layout (for runtime compilation) vs not.

I would like us to do the right thing here.